src/mg/PAMI/Message/Response/ResponseMessage.php

```diff
@@ -45,19 +46,19 @@
  * @license    http://marcelog.github.com/PAMI/ Apache License 2.0
  * @link       http://marcelog.github.com/PAMI/
  */
-class ResponseMessage extends IncomingMessage
+abstract class ResponseMessage extends IncomingMessage
```

Comment:
Why can't we leave this one as concrete, and just have other responses extend this one?